### PR TITLE
Add recipe for Goatools

### PR DESCRIPTION
**Pull request by bebatut**

### CODE REVIEW

File: recipes/goatools/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+build:
+  number: 0
+  skip: False
+source:
+  fn: goatools-0.5.9.tar.gz
+  md5: 2d5f1f9319dbf59850bf92cfa71e92ee
+  url: https://pypi.python.org/packages/source/g/goatools/goatools-0.5.9.tar.gz
+requirements:
+  build:
+    - python 2.7*
```
Comment (bgruening) [inline recipes/goatools/meta.yaml]: I guess a better way to do this is to just write `- python` here and above `skip: True # [py3k]`

Comment (bebatut) [inline recipes/goatools/meta.yaml]: Ok I will test like this

File: recipes/goatools/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  fn: goatools-0.5.9.tar.gz
+  md5: 2d5f1f9319dbf59850bf92cfa71e92ee
+  url: https://pypi.python.org/packages/source/g/goatools/goatools-0.5.9.tar.gz
+requirements:
+  build:
+    - python 
+    - setuptools
+    - fisher
+  run:
+    - python 2.7*
```
Comment (bgruening) [inline recipes/goatools/meta.yaml]: Still one python 2.7 missing :)

Comment (bebatut) [inline recipes/goatools/meta.yaml]: Oh, I missed it

File: recipes/goatools/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,26 @@
+about:
+    home: 'https://github.com/tanghaibao/goatools'
+    summary: "Python scripts to find enrichment of GO terms"
```
Comment (bgruening) [inline recipes/goatools/meta.yaml]: license is BSD, sorry have not spotted this earlier. If you submit your PR from the main branch I can fix those for you.

Comment (bebatut) [inline recipes/goatools/meta.yaml]: I had it :)

**Pull request merged**